build/config.gypi

```diff
@@ -0,0 +1,38 @@
+# Do not edit. File was generated by node-gyp's "configure" step
```

Comment:
This file should not be in the PR.  Please remove and add the file to .gitignore.